x/protocolpool/proto/cosmos/protocolpool/v1/tx.proto

```
@@ -3,6 +3,7 @@ package cosmos.protocolpool.v1;
 
 option go_package = "cosmossdk.io/x/protocolpool/types";
 
+import "cosmos/protocolpool/v1/types.proto";
```

buf

import "cosmos/protocolpool/v1/types.proto": file does not exist

(COMPILE)